modules/local/functions.nf

```diff
@@ -0,0 +1,79 @@
+//
```

Comment:
Why is this one recreated? It shouldn't be needed after the new module syntax (that was incorporated with template 2.3).